setup.py

```diff
@@ -39,12 +39,11 @@
     package_data={"": ["py.typed", "locale/*/LC_MESSAGES/*.mo"]},
     python_requires=">=3.6",
     install_requires=[
+        "pulp-glue>=0.17.0.dev,<0.18",
         "click>=8.0.0,<9.0.0",
-        "packaging",
```

Comment:
So contexts were/are the only things that rely on 'packaging' and 'requests', and that's now all in pulp_glue, correct?